commit 3284605dab0bb83e71c59f6da6356bef329a3f51
Author: Emery Hemingway <ehmry@posteo.net>
Date:   Sun Nov 3 15:06:04 2019 +0100

    Copy scripts to temporary dir, do not copy in-place
    
    This allows scripts to be executed in read-only Nimble package
    directories.

diff --git a/src/nimblepkg/nimscriptwrapper.nim b/src/nimblepkg/nimscriptwrapper.nim
index 4cfe6ec..805f7de 100644
--- a/src/nimblepkg/nimscriptwrapper.nim
+++ b/src/nimblepkg/nimscriptwrapper.nim
@@ -40,7 +40,7 @@ proc execNimscript(
   nimsFile, projectDir, actionName: string, options: Options, isHook: bool
 ): tuple[output: string, exitCode: int, stdout: string] =
   let
-    nimsFileCopied = projectDir / nimsFile.splitFile().name & "_" & getProcessId() & ".nims"
+    nimsFileCopied = getTempDir() / nimsFile.splitFile().name & "_" & getProcessId() & ".nims"
     outFile = getNimbleTempDir() & ".out"
 
   let
